.gitignore

```diff
@@ -1,6 +1,9 @@
 # Dependencies
 /node_modules
```

Comment:
調べてみましたが、分かりませんでした。
｢"/node_modules" gitignore｣で検索した上位10件のうち7割くらいが node_modules/ と書いていました。
なんでそういう書き方をするんですか？

Reply:
If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.
(from: https://git-scm.com/docs/gitignore )

これですかね？意思しか分からず、理由は不明ですが